source/containers/Form/hooks/formActions.ts

```diff
@@ -471,3 +471,42 @@ export function dirtyField(
   }
   return state;
 }
+/**
+ * Create snapshot of answers
+ * @param state 
+ */
```

Comment:
Noice words... the function name should tell me enought... please remove comments